### ISSUE TITLE
Make click-ability of headers visual

### ISSUE THREAD

**Issue by pitag-ha** [ocaml/v2.ocaml.org#1312]
This would be a **good first issue** for anyone looking to get started improving and contributing to OCaml.org.

There are several headers on different pages that are clickable. For example on Learn:

![Screenshot from 2021-03-29 19-21-05](https://user-images.githubusercontent.com/39061003/112874960-ddd9e800-90c3-11eb-84d0-31f3a1981eab.png)

In this extract, all of **Code Examples**, **Tutorials**, **FAQ** and **Books** can be clicked.

I suggest to go through all pages and to make clear everywhere which headers / header parts are clickable by giving them a link style.

**Comment by ChitvanRamani22** [comment]
Hi @patricoferris could you please assign this to me.

**Comment by patricoferris** [comment]
Hi @ChitvanRamani22 welcome to the community. 

Sure you can work on this. I've added some extra information in #1245 about the process. The summary is you don't need to be "Github assigned" to the issue.

**Comment by ChitvanRamani22** [comment]
@patricoferris okay thanks

**Comment by ChitvanRamani22** [comment]
@patricoferris i am a bit stuck and confused on how to install opam. Could you please help me.

**Comment by TildaDares** [comment]
@ChitvanRamani22 Follow this guide: https://ocaml.org/learn/tutorials/up_and_running.html

**Comment by ChitvanRamani22** [comment]
@TildaDares I referred this vedio https://youtu.be/xTsHMja8joA is it fine ?

**Comment by TildaDares** [comment]
@ChitvanRamani22 You should ask @patricoferris . I'm an outreachy applicant.

**Comment by ChitvanRamani22** [comment]
@TildaDares have you done the setup in your machine.

**Comment by TildaDares** [comment]
@ChitvanRamani22 Yeah I have. Do you have issues setting it up?

**Comment by pitag-ha** [comment]
@ChitvanRamani22, do you have a concrete question about how to install opam? Have you had a look at the [up and running guide](https://ocaml.org/learn/tutorials/up_and_running.html) @TildaDares has pointed you to? Is there anything concrete you don't understand about it?

**Comment by ChitvanRamani22** [comment]
@pitag-ha could you please help me in fixing this bug ![20210331_222208.jpg](https://user-images.githubusercontent.com/68965499/113181574-942e0080-926f-11eb-9de5-6a11a99f895d.jpg)

**Comment by pitag-ha** [comment]
In general, when you're asking for help with that kind of problems, it's very helpful to state which operating system you're on. Is it Windows? If so, have a look at the great new [windows troubleshooting section on CONTRIBUTING.md](https://github.com/ocaml/ocaml.org/pull/1343/files). If that doesn't solve your problem, it would be great if you could ask for help on a new issue as done e.g. [here](https://github.com/ocaml/ocaml.org/issues/1339).

**Comment by benny-coy** [comment]
> This would be a **good first issue** for anyone looking to get started improving and contributing to OCaml.org.
> 
> There are several headers on different pages that are clickable. For example on Learn:
> 
> ![Screenshot from 2021-03-29 19-21-05](https://user-images.githubusercontent.com/39061003/112874960-ddd9e800-90c3-11eb-84d0-31f3a1981eab.png)
> 
> In this extract, all of **Code Examples**, **Tutorials**, **FAQ** and **Books** can be clicked.
> 
> I suggest to go through all pages and to make clear everywhere which headers / header parts are clickable by giving them a link style.

**Comment by benny-coy** [comment]
I am an Outreachy applicant. I am new to open source and looking to get started improving and contributing to Ocaml.org. Please can I work on this issue.
Please I also need guidance in accomplishing task.
Thanks

**Comment by ChitvanRamani22** [comment]
> In general, when you're asking for help with that kind of problems, it's very helpful to state which operating system you're on. Is it Windows? If so, have a look at the great new [windows troubleshooting section on CONTRIBUTING.md](https://github.com/ocaml/ocaml.org/pull/1343/files). If that doesn't solve your problem, it would be great if you could ask for help on a new issue as done e.g. [here](https://github.com/ocaml/ocaml.org/issues/1339).

Yes it is windows . With lot of efforts i have reached here. But it has been 2 hours since this step (eval $(opam env))is running and not getting completed.![20210402_084433.jpg](https://user-images.githubusercontent.com/68965499/113376593-b323b400-938f-11eb-932f-9a1c467f6d12.jpg)

**Comment by pitag-ha** [comment]
@benny-coy, nice that you want to contribute to OCaml! Please have a look [here](https://github.com/ocaml/ocaml.org/issues/1245#issuecomment-809623832). We keep on opening new issues from time to time, but there are [a lot of applicants](https://github.com/ocaml/ocaml.org/issues/1245#issuecomment-810217825). Quite a few already open their own issues which is fantastic to see :)

**Comment by pitag-ha** [comment]
@ChitvanRamani22 , are you using WSL? [Here](https://github.com/ocaml/ocaml.org/pull/1343/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R69-R71) it says that it might be good to run `opam init --disable-sandboxing`. Did you do that or did you run `opam init`? Could you also show the output of `cat ~/.profile`? As mentioned before, it would be great if you could ask these questions in a separate issue. That way we keep this issue to what it is about and also for others it's easier to find the conversation about the set-up problem if they run into the same problem.

**Comment by ChitvanRamani22** [comment]
That error has when i closed and opened the terminal again but now i am facing this![IMG-20210402-WA0049.jpeg](https://user-images.githubusercontent.com/68965499/113406739-a2902f80-93c9-11eb-8d80-b94f2c964cdc.jpeg)

**Comment by patricoferris** [comment]
Let's keep the conversation of this particular issue over on https://github.com/ocaml/ocaml.org/issues/1366

**Comment by ChitvanRamani22** [comment]
@pitag-ha @patricoferris the setup is finally done and i have started working on the issue.
Just wanted to confirm that is this what we want as shown in the picture below.
![Screenshot_20210403-155831_WhatsApp.jpg](https://user-images.githubusercontent.com/68965499/113475881-77731200-9495-11eb-99c4-eaffee4893c9.jpg)

**Comment by patricoferris** [comment]
I can't speak for @pitag-ha but this looks like it is heading in the right direction, thanks @ChitvanRamani22 :)) I wonder if it would be better to style them like the other links on the page i.e. in the same orange/brown colour ?

**Comment by ChitvanRamani22** [comment]
Ok @patricoferris will do it in that way and might confirm once again thankyou so much for replying.

**Comment by gs0510** [comment]
Hi @ChitvanRamani22, did you make progress on this issue? Thanks!
